composer.json

```diff
@@ -33,9 +33,9 @@
         "psr/log": "^1.0",
         "swiftmailer/swiftmailer": "^6.1",
         "symfony/http-foundation": "^3.3",
-        "symfony/psr-http-message-bridge": "^1.0",
+        "symfony/psr-http-message-bridge": "~v1.1.0",
         "twig/extensions": "^1.5",
-        "twig/twig": "^2.5",
+        "twig/twig": "~v2.6.0",
```

Comment:
just using `"~2.6.0"` is imho more readable ;)

Reply:
…while the other one comes with `twig/twig` `v2.7.0`.